modules/environment/nix.nix

```diff
@@ -50,64 +50,25 @@ in
         '';
       };
 
-      ## From nix.nix
-
-      substituters = mkOption {
-        type = types.listOf types.str;
-        default = [ ];
-        description = ''
-          A list of URLs of substituters.  The official NixOS and Nix-on-Droid
-          substituters are added by default.
-        '';
-      };
-
-      trustedPublicKeys = mkOption {
-        type = types.listOf types.str;
-        default = [ ];
-        description = ''
-          A list of public keys.  When paths are copied from another Nix store (such as a
-          binary cache), they must be signed with one of these keys.  The official NixOS
-          and Nix-on-Droid public keys are added by default.
-        '';
-      };
-
-      extraOptions = mkOption {
-        type = types.lines;
-        default = "";
-        description = "Extra config to be appended to <filename>/etc/nix/nix.conf</filename>.";
-      };
     };
 
   };
 
 
   ###### implementation
 
-  config = mkMerge [
-    (mkIf cfg.enable {
-      environment.etc = {
-        "nix/nix.conf".text = ''
-          sandbox = false
-          substituters = ${concatStringsSep " " cfg.substituters}
-          trusted-public-keys = ${concatStringsSep " " cfg.trustedPublicKeys}
-          ${cfg.extraOptions}
-        '';
-      };
-    })
- 
-    {
-      nix = {
-        enable = true;
-        substituters = [
-          "https://cache.nixos.org"
-          "https://nix-on-droid.cachix.org"
-        ];
-        trustedPublicKeys = [
-          "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY="
-          "nix-on-droid.cachix.org-1:56snoMJTXmDRC1Ei24CmKoUqvHJ9XCp+nidK7qkMQrU="
-        ];
-      };
-    }
-  ];
+  config = {
+    nix = {
+      enable = true;
```

Comment:
so, you're introducing .enable, but not using it?

Reply:
Yes. I just updated this PR to emphasize this.